src/Attachment.tsx

```diff
@@ -170,34 +169,44 @@ export const AttachmentView = (props: {
         case "application/vnd.microsoft.card.hero":
             if (!attachment.content)
                 return null;
+
+            const heroCardBuilder = new CardBuilder.AdaptiveCardBuilder();
```

Comment:
I don't know that I'm convinced. :-) But I'm not going to make this a showstopper.